scripts/earthly-cloud

```diff
@@ -60,17 +60,20 @@ if $INTERACTIVE ; then
   # don't play nice with tee if interactive
   earthly $EARTHLY_FLAGS $@
 elif ! earthly $EARTHLY_FLAGS $@ 2>&1 | tee .output.txt >&2 ; then
-  # we got a failure, handle retries
+  # we try earthly once, capturing output
+  # if we get one of our (unfortunate) known failures, handle retries
   # TODO potentially handle other intermittent errors here
   if grep 'failed to get edge: inconsistent graph state' .output.txt >/dev/null ; then
-    # TODO we need to limit earthly parallelism or sometimes get
+    # TODO when earthly is overloaded we sometimes get 
     # 'failed to solve: failed to get edge: inconsistent graph state'
-    # so we use arbitrary WAIT statement groups. Pending earthly support.
-    # for now we just try to recover from the failures.
     echo "Got 'inconsistent graph state'. Restarting earthly. See https://github.com/earthly/earthly/issues/2454'"
     earthly $EARTHLY_FLAGS $@
+  elif grep 'unexpected content-type' .output.txt >/dev/null ; then
```

Comment:
I'm a bit worried we could be accidentally logging this in our own tests (due to an error) and this would lead to an infinite recursion enqueuing the job again.  Do you think we can narrow it down, or at least set a maximum number of retries?

Reply:
That makes sense

Reply:
this is just one retry though

Reply:
left alone until seen in the wild again

Reply:
> this is just one retry though

I thought it reentered the same script. Nevermind then!